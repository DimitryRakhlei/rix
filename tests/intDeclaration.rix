#a = 1
#b = 0
#c = -1
#temp = c
c = b
b = a
a = temp
#d = (#e = 2)
#f = 4294967296

print ("a = " + a)
print ("b = " + b)
print ("c = " + c)
print ("d = " + d)
print ("e = " + e)
print ("f = " + f)
