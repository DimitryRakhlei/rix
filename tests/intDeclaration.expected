a = -1
b = 1
c = 0
d = 2
e = 2
f = 0
